VIDEO_3_SCRIPT.md

```diff
@@ -0,0 +1,58 @@
+
+AsyncAPI was founded by Fran Mendez, the Specification started as a side project at Hitch (API Changelog). He identified a lack of tools in the message-driven space and searched for a solution for standardized message-driven specification systems. 
+
+ ## What AsyncAPI brings
+
+Its main contribution to the world of APIs is the standardization of how you can design and describe Event-Driven Architectures in a simplified and unified manner.
+
+We could even say that AsyncAPI is a specification that facilitates, simplifies, and accelerates the work of developers working with APIs. Having defined, documented, and accepted the communication between APIs, AsyncAPI provides an increased and more efficient information control.
+
+AsyncAPI Initiative also develops and maintains an extensive amount of AsyncAPI tools that accelerate application development. The tools' mechanisms for automating complex and essential tasks facilitate code generation, documentation generation, validators, etc. 
+
+## What is AsyncAPI Initiative and what are its objectives
+
+As an open source initiative, AsyncAPI seeks to improve the current state of Event-Driven Architectures (EDA). Its long-term goal is to make working with EDAs as easy as it's to work with REST APIs. That goes from documentation to code generation, from discovery to event management.
+
+AsyncAPI Initiative currently aims to accomplish three main goals:
+
+1. Unify all API specifications. No one does event-driven architecture alone. Most developers complement the event architecture with REST (OpenAPI), GraphQL, and RPC APIs. Let us embrace reusability!
+
+2. Make the development experience super seamless, from idea to production. Users should be able to create their first API in a matter of minutes without prior knowledge of AsyncAPI.
+
+3. Reach the first position among the most used API specifications. This will require a growth in the number of community members by ensuring it is a community-driven project running under open governance.
+
+## AsyncAPI foundations
+
+An important step the initiative took towards its consolidation was to partner with Postman at the end of 2020, a leading collaborative platform developing APIs. This partnership allows investment in a dedicated department working full time on AsyncAPI spec, tools and community-building aspects.
+
+The strategy is to continue improving, implementing resources that make it more complete and solid. This has led to an agreement of joinment between AsyncAPI Initiative and the Linux Foundation in 2021. As a result AsyncAPI Initiative is fostered and hosted by a neutral foundation that guarantees the long-term sustainability and success of the project. 
```

Comment:
`joinment` - afaik it means we kind of joined them in a way like a "merge". But we did not. Proper term is that "host". So something like: 

```suggestion
The strategy is to continue improving, implementing resources that make it more complete and solid. This has led to an agreement to host AsyncAPI under the Linux Foundation. As a result AsyncAPI Initiative is fostered and hosted by a neutral foundation that guarantees the long-term sustainability and success of the project. 
```